main.py

```diff
@@ -1,16 +1,27 @@
-# This is a sample Python script.
```

Comment:
Please, create a requirements file ( you may read about 'pip freeze' to create it)

Reply:
Done